recipe/meta.yaml

```diff
@@ -117,6 +117,8 @@ outputs:
         - llvm-openmp                         # [osx]
         - libgomp                             # [linux]
       host:
+        - llvm-openmp                         # [osx]
```

Comment:
Don't the strong run_exports cover this already?

Reply:
Explained in the PR comment and https://github.com/conda-forge/openmp-feedstock/issues/126#issuecomment-2198847141